distributions/rhel-95/rhel-9.6

```diff
@@ -0,0 +1 @@
+rhel-9.6
```

Comment:
This looks like a symlink, since the numbers do not match I wonder if this is some leftover?

Reply:
ah yes, that's not supposed to be there